lang/py3/avro/schema.py

```diff
@@ -385,7 +385,7 @@ def NewWithDefaultNamespace(self, namespace):
     """
     return Names(names=self._names, default_namespace=namespace)
 
-  def GetName(self, name, namespace=None):
+  def _get_name(self, name, namespace=None):
```

Comment:
I wasn't 100% sure on this one, because `get_name` is already a function. When I did a usage search for `GetName`, the usages looked to be all within this file. When I searched for `get_name`, it seemed like other modules were using it.

Being that the existing `get_name` function delegates to this one, I think the functionality is still available for external use.

Does that seem appropriate to you? What are your thoughts?